app/models/manageiq/providers/openstack/inventory/parser/cloud_manager.rb

```diff
@@ -281,7 +281,13 @@ def orchestration_stacks
       o.status_reason = stack.stack_status_reason
       o.parent = persister.orchestration_stacks.lazy_find(stack.parent)
       o.orchestration_template = orchestration_template(stack)
-      o.cloud_tenant = persister.cloud_tenants.lazy_find(stack.service.current_tenant["id"])
+      # stack parameters can miss tenant_id, so we make admin default tenant
+      tenant_id = if stack.parameters
```

Comment:
You should do something roughly like `if stack.parameters && stack.parameters["OS::project_id"]`, because this current conditional will return nil if `stack` has a value for `parameters`, but doesn't have a value for `OS::project_id`